internal/presenter/packages/spdx_helpers.go

```diff
@@ -87,6 +87,13 @@ func getSPDXLicense(p *pkg.Package) string {
 	for _, l := range p.Licenses {
 		if value, exists := spdxlicense.ID(l); exists {
 			parsedLicenses = append(parsedLicenses, value)
+			continue
+		}
+
+		// we did not find an explicit spdx license
+		// check congruent debian mappings
+		if value, exists := spdxlicense.DebianID(l); exists {
```

Comment:
in this case we'd be potentially mapping the license as if it were debian related without verifying that it is debian related.

Reply:
Yea - this stems from the other comment about not having too specific a function. If I wrap this all into the `spdxlicense.ID` function where we first check the `go:generate` list, then we can just update the naming so that our second check isn't debian specific, but more of a check for incongruent mappings as they appear in the SPDX spec.

Reply:
@wagoodman cleaned up the interface so that we are just using ID now. Map has been updated to known short name exceptions so it can be expanded to include other oddities that might be found in other disto.